hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -696,7 +698,7 @@ public S3AInputPolicy getInputPolicy() {
    * @return the change detection policy
    */
   @VisibleForTesting
```

Comment:
No longer needs @VisibleForTesting annotation now, since public

Reply:
I actually thought a little about this.  I was thinking of `@VisibileForTesting` in this case as documenting this method is only public to allow access in tests (in a different package).  I know it is more typically used on protected or package-private methods.

I'm curious if there is any feedback about this being public in general?  I'm accessing it across packages in a couple of tests (in the s3guard and select packages).

I can reinforce that it is only visible for tests by mentioning that explicitly in the javadoc.

Reply:
See updated javadoc to mention only public to allow access in tests in other packages.